.circleci/config.yml

```diff
@@ -94,6 +94,13 @@ jobs:
             find . -type f -regex  ".*/test-results.*\.xml" -exec cp {} $CIRCLE_TEST_REPORTS/ \;
           when: always
 
+      - run:
+          name: Build Notifications Panel
+          command: |
+            if [ "$CIRCLE_NODE_INDEX" == "0" ]; then
```

Comment:
This is a 26s job in the CI run on this PR which would run for every CI job. Should we match `"$CIRCLE_BRANCH" == "master"` as well?

I'm not familiar with the development flow of notifications panel. Will these artifacts be needed for development? In that case, the dev would have to commit to swap `master` for the branch name so produce the artifact while working on this.

We could use a different node than `0` for this, we already build translation artifacts on `0`, but I hope to be running independent, parallel jobs soon (#27180). This would be a good candidate for an independent job.

Reply:
> Should we match "$CIRCLE_BRANCH" == "master" as well?

This is something I thought about and if we limit it to master then we make it hard to test out. We'd want to make sure that our proposed changes work in the iframe version before merging is my estimate.

> Will these artifacts be needed for development?

Yeah, and I hope we have some tests built up with it too so that we can do extra checks like making sure that changes to Calypso code don't unintentionally destroy the iframe version of the notifications panel.

> I hope to be running independent, parallel jobs soon (#27180). This would be a good candidate for an independent job.

Exactly! 🎉 

> We could use a different node than 0 for this

Do you have a suggestion for a different node at the moment? Where could we put this so that it doesn't delay the existing test suite?

Reply:
>  devs who want to trigger a notifications build would click a button in the CircleCI UI

Fancy ✨ That'd be great